frontend/src/app/components/wp-relations/wp-relations-create/wp-relations-autocomplete/wp-relations-autocomplete.component.ts

```diff
@@ -113,9 +137,8 @@ export class WorkPackageRelationsAutocomplete implements AfterContentInit {
     if (!this.ngSelectComponent) {
       return;
     }
-
     setTimeout(() => {
-      this.ngSelectComponent.focus();
+       this.ngSelectComponent.ngSelectInstance.focus();
```

Comment:
A setTimeout triggers ChangeDetection for the entire app which could hit performance. To avoid that, it is recommended to place the setTimeout inside ngZone.runOutsideAngular.

Nevertheless, I think the root issue is in the OpAutocompleterComponent, it should provide the focus functionality from inside, so the user only has to call this.ngSelectComponent.focus():

`
this.ngZone.runOutsideAngular(() => {
      setTimeout(() => {        
        this.ngSelectInstance.focus();
      });
 });     
`

Why 25 milliseconds? Relying on concrete timings is usually error-prone.